runtime/http_server_apis.md

```diff
@@ -174,7 +174,7 @@ function handler(req: Request): Response {
   const upgrade = req.headers.get("upgrade") || "";
   let socket, response;
   try {
-    res = Deno.upgradeWebSocket(req);
+    const res = Deno.upgradeWebSocket(req);
```

Comment:
Actually shouldn't it be `response = ` so it is available outside the try block? Looks like both are wrong.